src/main/kotlin/domain/Game.kt

```diff
@@ -0,0 +1,27 @@
+package domain
+
+import kotlin.random.Random
+
+class Game {
+    var carList = mutableListOf<RacingCar>()
```

Comment:
```suggestion
    val carList = ListOf<RacingCar>()
```

자동차 참여자는 한번 만들어지면, 변경이 필요 없어보여요. carList를 불변으로 만들어볼까요?

Reply:
List 값으로 받게 만들었습니다~